spec/Candid.md

```diff
@@ -144,10 +144,12 @@ A *service* is a standalone actor on the platform that can communicate with othe
 
 #### Structure
 
-A service's signature is described by an *actor type*, which defines the list of *methods* that the service provides. Each method is described by its *name* and a *function type* describing its signature. The function type can also be given by referring to a type definition naming a function reference type.
+A service's signature is described by an *actor type*, which defines the list of *methods* that the service provides. 
+A service can optionally provide *initialization parameters* for creating services dynamically with the same signature.
+Each method is described by its *name* and a *function type* describing its signature. The function type can also be given by referring to a type definition naming a function reference type.
```

Comment:
This sentence elaborates on the one above, so should probably stay directly after.